src/player-wrapper.js

```diff
@@ -515,13 +515,15 @@ PlayerWrapper.prototype.onAdStart = function() {
  */
 PlayerWrapper.prototype.onAllAdsCompleted = function() {
   if (this.contentComplete == true) {
-    if (this.h5Player.src != this.contentSource) {
-      // Avoid setted autoplay after the post-roll
-      this.vjsPlayer.autoplay(false);
-      this.vjsPlayer.src({
-        src: this.contentSource,
-        type: this.contentSourceType,
-      });
+    if ((typeof this.h5Player.src === 'string') && (this.h5Player.src.indexOf('blob:') === -1) ) {
```

Comment:
h5Player is initialised when onPlayerReady(), the videojs may not set source at that time, so h5Player.src is an empty string. 
Here i think we should check vjsPlayer.src !== this.contentSource